From ef6473e45dc5c769040ddd0ac42dc9bb1f271512 Mon Sep 17 00:00:00 2001
From: Michael H <michael@michaelhall.tech>
Date: Thu, 28 Apr 2022 06:30:21 +0000
Subject: [PATCH] lazy

---
 setup.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/setup.py b/setup.py
index dc8de9e..93a79eb 100644
--- a/setup.py
+++ b/setup.py
@@ -442,7 +442,7 @@ class fetch(Command):
 # We allow enable/omit to be specified to build and then pass them to build_ext
 build_enable = None
 build_omit = None
-build_enable_all_extensions = False
+build_enable_all_extensions = True

 bparent = build.build

@@ -459,7 +459,7 @@ class apsw_build(bparent):
         v = bparent.initialize_options(self)
         self.enable = None
         self.omit = None
-        self.enable_all_extensions = build_enable_all_extensions
+        self.enable_all_extensions = True
         return v

     def finalize_options(self):
@@ -503,7 +503,7 @@ class apsw_build_ext(beparent):
         v = beparent.initialize_options(self)
         self.enable = build_enable
         self.omit = build_omit
-        self.enable_all_extensions = build_enable_all_extensions
+        self.enable_all_extensions = True
         return v

     def finalize_options(self):
--
2.25.1
