From 7b2bd81c8e062e7c557885aa9f833e97b8092cb0 Mon Sep 17 00:00:00 2001
From: Michael H <michael@michaelhall.tech>
Date: Sat, 20 Mar 2021 19:27:28 -0400
Subject: [PATCH] .

---
 checksums | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/checksums b/checksums
index 631a104..f0292dd 100644
--- a/checksums
+++ b/checksums
@@ -35,3 +35,5 @@ https://sqlite.org/2021/sqlite-autoconf-3340100.tar.gz 2930089 c20286e11fe5c2e37
 https://sqlite.org/2021/sqlite-amalgamation-3350000.zip 2457978 ba64bad885c9f51df765a9624700747e7bf21b79 f5d0c859bf152415a047c13068748991
 https://sqlite.org/2021/sqlite-autoconf-3350000.tar.gz 2955394 360b4356f6a80a4d2b14834629e387d0fa3aaa3b 1b51302322aef8aec70f129147e666b3
 
+https://sqlite.org/2021/sqlite-amalgamation-3350200.zip 2458191 5d5461e0eb229aee52b0ee330124b4ce5edb66a1 732c5d0758a2a2fb9e5b9d6224141a01
+https://sqlite.org/2021/sqlite-autoconf-3350200.tar.gz 2955661 ba6994db6d4ffd2797b9340aeb4ad8bd616176f8 454e0899d99a7b28825db3d807526774
-- 
2.28.0