From e72304f9398f0b32add6960e3e2850e2543d291c Mon Sep 17 00:00:00 2001
From: Michael H <michael@michaelhall.tech>
Date: Fri, 2 Apr 2021 13:29:24 +0000
Subject: [PATCH 2/2] hashes

---
 checksums | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/checksums b/checksums
index 5bfb9b8..657af1e 100644
--- a/checksums
+++ b/checksums
@@ -32,6 +32,9 @@ https://sqlite.org/2020/sqlite-autoconf-3340000.tar.gz 2930085 1544957cf4bcc9606
 https://sqlite.org/2021/sqlite-amalgamation-3340100.zip 2433624 adf5f5f2b859a9fcbf5fdf6f87b2cd42949c267c fc92d9275d9baa483c7199f44b86d4ae
 https://sqlite.org/2021/sqlite-autoconf-3340100.tar.gz 2930089 c20286e11fe5c2e3712ce74890e1692417de6890 14dca9d44fb6a57f6521c1ace4be5a68

+
+https://sqlite.org/2021/sqlite-autoconf-3350300.tar.gz 2956098 d592a0c3ecc28bdb4169b3f108b7ee4b9c675f3b cadd4db9b528dfd0a7efde39f767cd83
+
 https://sqlite.org/2021/sqlite-amalgamation-3350000.zip 2457978 ba64bad885c9f51df765a9624700747e7bf21b79 f5d0c859bf152415a047c13068748991
 https://sqlite.org/2021/sqlite-autoconf-3350000.tar.gz 2955394 360b4356f6a80a4d2b14834629e387d0fa3aaa3b 1b51302322aef8aec70f129147e666b3
 https://sqlite.org/2021/sqlite-amalgamation-3350200.zip 2458191 5d5461e0eb229aee52b0ee330124b4ce5edb66a1 732c5d0758a2a2fb9e5b9d6224141a01
--
2.30.0
